### DISCUSSION TITLE
[zksync-cli] Failed to clone template

### DISCUSSION THREAD

**Discussion by shikicloud** [zkSync-Community-Hub/zksync-developers#608] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

I cannot create a template while using "npx zksync-cli create":
? Enter a folder name for your project: my-zksync-project
? What type of project do you want to create? Contracts
? Ethereum framework Ethers v6
? Template Hardhat + Solidity
? Private key of the wallet responsible for deploying contracts (optional) 
****************************************************************
? Package manager npm

Setting up template in /Users//Desktop/zksync tutorial/my-zksync-project...
✖ Failed to clone template
ⓘ There was an error while creating new project:
ⓘ Command exited with code 129: (fatal: Too many arguments.)

**Comment by JackHamer09** [comment, marked as answer]
@shikicloud please make sure target location doesn't include any spaces in the path

**Reply by shikicloud** [reply]
Thank you!
